\begin{titlepage}
\begin{center}

	% Title
	{ \huge \bfseries Assemble yourself  \\[0.4cm] }
	A printable bioinformatics boardgame


\end{center}
{\large
Assemble yourself is a paper-printable bioinformatics boardgame which is designed to be educational, fun and engaging. 
It introduces basic genetics and Next Generation Sequencing through a a puzzle-like problem. 

The main idea of the game is to manually perform a assembly of a genome sequence
from next generation reads. The entire game is printed on paper. This includes the reads (cut out paperstrips) 
and a board which serves scaffold to assemble the reads and writing down the sequence.
Once the reads have been assembled and a consensus sequence is found, the consensus
sequence is translated into amino acid sequences in both DNA strands. The letters
representing the amino acid sequence contains a secret word. Once the player
finds the secret word he wins the game. 

The game is generated by a computer program and many aspects of the game can 
be configured prior to generation. For instance, the secret protein word
can be changed and the minimal read depth at any loci can be configured. 

The game can be played by a single person or run as a competion, e.g.,  in a class. I have experienced
the latter scenario to be a lot of fun, especially when students work in small groups. 

To get the most out the game, players should have been introduced to/be familiar with the central dogma of biology
and concepts like DNA, base-pairing, amino acid, strand, reading frame, gene and codon before playing the game.
}

\vfill 
{\small
Copyright (C) 2014 Christian Theil Have

This program is free software: you can redistribute it and/or modify
it under the terms of the GNU General Public License as published by
the Free Software Foundation, either version 3 of the License, or
(at your option) any later version.

This program is distributed in the hope that it will be useful,
but WITHOUT ANY WARRANTY; without even the implied warranty of
MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
GNU General Public License for more details.

You should have received a copy of the GNU General Public License
along with this program.  If not, see \verb|<http://www.gnu.org/licenses/>|.
}

\end{titlepage}

\section*{Introduction to the game}

Recently an interesting protein with the amino acid sequence \mutantprotein{ }was found in the bacteria \emph{S. Equencia}. 
It is now to be determined if a homologue exists in the species \emph{B. Ionformatica}.

To determine this a lab amplificied a relevant part of the DNA of \emph{B. Ionformatica} using PCR primers flanking
the gene in \emph{S. Equencia} which are believed to be highly conserved also in \emph{B. Ionformatica}, allthough
the sequence of \emph{B. Ionformatica} is currently not known.
The amplified DNA was sequenced using Ullamini LoSeq next generation sequencing technology yielding \numberofreads{ } reads.
The quality of the reads are not perfect -- read errors resulting in random read ``mutations`` are expected in one out of twenty bases. 

As a bioinformatician you are given the task to find out if \emph{B. Ionformatica} has a homologue of the protein \mutantprotein{ } and 
determine how its amino acid sequence differs in \emph{B. Ionformatica}.
However, the high performance moon grid engine supercluster is currently down (as it sometimes is) and you have to do it all by hand. 
Fortunely, you have printed all the reads. 
You task is as follows:

\begin{itemize} 
\item Perform de-novo assembly of all the reads
\item Find open reading frames that may contain a gene
\item Find the amino acid sequence of any such gene to determine if it could be a homologue to \mutantprotein{ }
\item Report your finding and claim eternal fame 
\end{itemize}

But you have to hurry! Many other competing research groups have also gotten a hold of the reads and 
they will scoop you on this important discovery if you are not fast. 

\subsection*{Detailed instructions}

Cut the reads with a scissor so you have them as paper strips. 
Place these paper strips horizontally on the board in the read alignment area so that they overlap each other, matching base by base. 
Above the read alignment area is a row which have six predefined nucleotides in each side (the PCR primers). 
Your reads should also match these.  
Once you have aligned all the reads, you can fill the empty cells with the consensus sequence obtained from the alignment. 
It is possible to map all the reads, but not all reads may not map to the same strand. 
If you end up with reads that do not fit anywhere, then most likely your alignment is wrong. 

Once you have the entire  consensus sequence you look for the hidden protein. 
In the three top of the board you can write down amino acids  (and start and stop codons) for the forward strand corresponding to your consensus sequence. 
In the bottom you do the same for the reverse strand.

An open reading frame consist of a start codon, some intermediate codons and a stop codon. Identify open reading frames and translate intermediate codons using the supplied 
amino acid table. Find the protein that looks the most homologue. 

\paragraph{Winning the game:}
The team that first correctly reports the amino acid sequence\footnote{Report amino acid sequence without the methionine encoded by the start codon} of the protein homologue and also its nucletide sequence wins the game.

\paragraph{Feedback during the game:} 
During the game a team can hand in the IUPAC consensus codes of their alignment (see attached table).
The judge of the game will then determine if the sequence is correct. Similarly you may make hand it 
a protein sequence and the judge will check its correctness (if correct, you win).  However, this service comes at a price.
The first time you ask it will cost you one of your reads. The second time it cost you two reads, the third time four reads and so on. 
So, use your reads wisely.
