\section*{Introduction to the game}

Recently an interesting protein with the amino acid sequence \mutantprotein{ }was found in the bacteria \emph{S. Equencia}. 
It is now to be determined if a homologue exists in the species \emph{B. Ionformatica}.

To determine this a lab amplificied a relevant part of the DNA of \emph{B. Ionformatica} using PCR primers flanking
the gene in \emph{S. Equencia} which are believed to be highly conserved also in \emph{B. Ionformatica}, allthough
the sequence of \emph{B. Ionformatica} is currently not known.
The amplified DNA was sequenced using Ullamini LoSeq next generation sequencing technology yielding \numberofreads{ } reads.
The quality of the reads are not perfect -- read errors resulting in random read ``mutations`` are expected in one out of twenty bases. 

As a bioinformatician you are given the task to find out if \emph{B. Ionformatica} has a homologue of the protein \mutantprotein{ } and 
determine how its amino acid sequence differs in \emph{B. Ionformatica}.
However, the high performance moon grid engine supercluster is currently down (as it sometimes is) and you have to do it all by hand. 
Fortunely, you have printed all the reads. 
You task is as follows:

\begin{itemize} 
\item Perform de-novo assembly of all the reads
\item Find open reading frames that may contain a gene
\item Find the amino acid sequence of any such gene to determine if it could be a homologue to \mutantprotein{ }
\item Report your finding and claim eternal fame 
\end{itemize}

But you have to hurry! Many other competing research groups have also gotten a hold of the reads and 
they will scoop you on this important discovery if you are not fast. 

\subsection*{Detailed instructions}

Cut the reads with a scissor so you have them as paper strips. Place these paper strips horizontally on the board in the read alignment area
 so that they overlap each other, macthing base by base. Note that they may not match perfectly. Above the read alignment area
is a row which have six predefined nucleotides in each side (the PCR primers). Your reads should also match these.  
Once you have aligned all the reads, you can fill the empty cells with the consensus sequence obtained from the alignment. 
It is possible to map all the reads and all reads map to the same strand. 
If you end up with reads that do not fit anywhere, then most likely your alignment is wrong. 

Once you have the entire  consensus sequence (or perhaps just a part of it) you can begin to look for the protein. In the three top most rows
of board you can write down amino acids and start and stop codons corresponding to the consensus sequence. An open reading frame consist of 
a start codon, some intermediate codons and a stop codon. Identify open reading frames and translate intermediate codons using the supplied 
amino acid table. Find the protein that looks the most homologue. 

\paragraph{Winning the game:}
The team that first correctly reports the amino acid sequence\footnote{Report amino acid sequence without the methionine encoded by the start codon} of the protein homologue and also its nucletide sequence wins the game.

\paragraph{Feedback during the game:} 
During the game a team can hand in the IUPAC consensus codes of their alignment (see attached table).
The judge of the game will then determine if the sequence is correct. Similarly you may make hand it 
a protein sequence and the judge will check its correctness (if correct, you win).  However, this service comes at a price.
The first time you ask it will cost you one of your reads. The second time it cost you two reads, the third time four reads and so on. 
So, use your reads wisely.

